src/shared/scrapers/USA/MA/index.js

```diff
@@ -114,6 +119,56 @@ const scraper = {
 
       counties.push(summedData);
 
+      return geography.addEmptyRegions(counties, this._counties, 'county');
+    },
+    '2020-3-30': async function() {
+      this.type = 'json';
+      this.url =
+        'https://services1.arcgis.com/TXaY625xGc0yvAuQ/arcgis/rest/services/COVID_CASES_MA/FeatureServer/0/query?f=json&where=1%3D1&returnGeometry=false&outFields=*';
+      const counties = [];
+      const data = await fetch.json(this.url);
+
+      let onlySumDeaths = true;
+      let sumDeaths = 0;
+
+      data.features.forEach(item => {
+        const cases = item.attributes.CASES;
+        const county = geography.addCounty(
+          item.attributes.County.charAt(0) + item.attributes.County.slice(1).toLowerCase()
+        );
+
+        if (datetime.scrapeDateIsBefore(item.attributes.EditDate)) {
```

Comment:
Is this logic right? Don't you want a NOT in front here...

if ( ! datetime.scrapeDateIsBefore(item.attributes.EditDate)) { throw ... }

Reply:
It is. It confused me for a while too because I think the function naming is backwards, but I manually tested it with different dates and it worked